### DISCUSSION TITLE
Can I only port its stylesheets to other front-end projects?

### DISCUSSION THREAD

**Discussion by LittleYe233** [material-components/material-web#5747] [Q&A]
Maybe it is a weird question here, but I want to use the stylesheets for supported components for other projects. For example, is there any equivalent style configuration for a simple `<button>` element with a specific class with minimum imported JavaScript modules? Like this to get a filled button:

```html
<!-- link some stylesheet file -->
<!-- import least number of JavaScript modules -->
<button class="my-filled-button" other-needed-attr="value"></button>
```

Thanks for your generous help.

**Comment by asyncliz** [comment]
The only classes MWC provides for use [are for typography](https://github.com/material-components/material-web/blob/main/docs/theming/typography.md#classes). There aren't component classes intended to style elements like a `<button>` to look like Material.

You'd need to use Sass and `@material/web/tokens` to create classes that apply the component token styles manually.
